### PR TITLE
feat(autocomplete): Added support for joined tables and using autocomplete on join constr…

### PR DESCRIPTION
**Pull request by Gilbert09**
## Changes
- Added support for using autocomplete on joined tables and on the join constraint

image

image

## How did you test this code?
- unit tests

### CODE REVIEW

File: bin/start [hunk 1]
```diff
@@ -18,7 +18,7 @@ curl -s 'http://localhost:8123/ping' || ( service_warning 'ClickHouse'; bin/chec
 [ ! -f ./share/GeoLite2-City.mmdb ] && ( curl -L "https://mmdbcdn.posthog.net/" | brotli --decompress --output=./share/GeoLite2-City.mmdb )
 
 ./bin/start-worker &
-./bin/start-backend &
+# ./bin/start-backend &
```
Comment (mariusandra) [inline bin/start:21]: 🤔

File: posthog/clickhouse/client/execute_async.py [hunk 2]
```diff
@@ -115,7 +115,7 @@ def enqueue_process_query_task(
     query_json,
     query_id=None,
     refresh_requested=False,
-    bypass_celery=False,
+    bypass_celery=True,
```
Comment (mariusandra) [inline posthog/clickhouse/client/execute_async.py:118]: 🤔

File: posthog/settings/data_stores.py [hunk 3]
```diff
@@ -288,7 +288,7 @@ def _parse_kafka_hosts(hosts_string: str) -> List[str]:
 
 CACHES = {
     "default": {
-        "BACKEND": "django_redis.cache.RedisCache",
+        "BACKEND": "django.core.cache.backends.dummy.DummyCache",
```
Comment (mariusandra) [inline posthog/settings/data_stores.py:291]: 🤔

**Review by mariusandra — COMMENTED** [review]
I see some seemingly unrelated changes

File: posthog/settings/data_stores.py (hunk 3 above)
Comment (Gilbert09) [reply posthog/settings/data_stores.py:291]: Oh poop, none of these were meant to get committed 🤦  thank you

Comment (mariusandra) [reply posthog/settings/data_stores.py:291]: Now that's some foul language 🤣

Comment (Gilbert09) [reply posthog/settings/data_stores.py]: All fixed

**Review by mariusandra — APPROVED** [review]

**Pull request merged**